src/css/custom.css

```diff
@@ -53,6 +54,52 @@ html[data-theme='dark'] .header-github-link:before {
     no-repeat;
 }
 
+.website-link:before {
```

Comment:
can you move that part and below into your `src/components/ProfileCard/styles.module.css`?

Reply:
sure. I was going to use it the icons in the footer but I can add it here and when I make footer changes I can move it back to global if needed